Makefile

```diff
@@ -1,6 +1,6 @@
 PROJECT = contour
 MODULE = github.com/heptio/$(PROJECT)
-REGISTRY ?= gcr.io/heptio-images
+REGISTRY ?= projectcontour
```

Comment:
is it worth being concrete here; `docker.io/projectcontour` ?

Reply:
it won't hurt anything, I guess. I don't see the assumption that 'no domain == docker hub' changing now though.